### DISCUSSION TITLE
Useless commands

### DISCUSSION THREAD

**Discussion by Doges** [SwitchbladeBot/switchblade#1297] [General]
Should we remove some useless commands like `s!bolinhadegorfe`? bruh, the meme died 1-2yrs ago

**Comment by pedrofracassi** [comment]
remove nsfw stuff

**Reply by Doges** [reply]
fuck you

**Reply by pedrofracassi** [reply]
no

**Reply by Doges** [reply]
alright but where's the dashboard to disable nsfw commands :hahaa:

**Comment by moruzerinho6** [comment]
ye

**Comment by davipatury** [comment]
selling used celta 2009 without spare tire

**Reply by Doges** [reply]
price?

**Comment by metehus** [comment]
slasjhc oamndos
